src/sentry/integrations/bitbucket/integration.py

```diff
@@ -95,8 +95,8 @@ def get_repositories(self, query=None):
                 for repo in resp.get("values", [])
             ]
 
-        exact_query = f'name="{query}"'.encode()
-        fuzzy_query = f'name~"{query}"'.encode()
+        exact_query = f'name="{query}"'
```

Comment:
you're changing the type of this variable, are we sure that is safe and not just broken typing further down the road? assuming the code currently works fine

Reply:
yes that's intentional and part of the change -- I surveyed all callsites and the `bytes` here was incorrect (it was being coerced to bytes and then out of bytes for no reason) -- these are also fairly well covered by tests